drivers/gpio/gpio_stm32.c

```diff
@@ -163,6 +163,10 @@ int gpio_stm32_configure(u32_t *base_addr, int pin, int conf, int altf)
 		}
 	}
 
+#if defined(CONFIG_SOC_SERIES_STM32L4X) && defined(GPIO_ASCR_ASC0)
+	LL_GPIO_EnablePinAnalogControl(gpio, pin_ll);
```

Comment:
Should this be done for every pin, and any configuration?
I mean, if I configure a led or an uart pin, I'll run into this code, is this correct?

Reply:
Sorry, this is wrong, my mistake... should be only configured in analog mode.